packages-next/types/src/config/lists.ts

```diff
@@ -46,29 +49,29 @@ export type ListConfig<
    */
   description?: string; // defaults both { adminUI: { description }, graphQL: { description } }
 
-  // Not currently supported
-  // plugins?: any[]; // array of plugins that can modify the list config
-
   /**
    * The label used for the list
    * @default listKey.replace(/([a-z])([A-Z])/g, '$1 $2').split(/\s|_|\-/).filter(i => i).map(upcase).join(' ');
    */
-  // Not currently supported
-  // label?: string;
+  label?: string;
```

Comment:
Do we support these new config options now? If so we should make sure these are documented in `schema.mdx` now before we forget about them.

Reply:
I've just removed support for them for now so we can make sure they're in the place we want them in a future PR.